sys/include/net/netopt.h

```diff
@@ -202,6 +202,13 @@ typedef enum {
     NETOPT_CCA_THRESHOLD,
 
     /**
+     * @brief CCA mode for the radio transceiver
+     */
+    NETOPT_CCA_MODE,            /**< get/set the CCA mode as uint8_t
+                                 *   corresponding to the respective PHY standard,
+                                 *   e.g Std 802.15.4 */
```

Comment:
place it in the drivers/include/net/netdev2/ieee802154.h , if it is wrong, can you please suggest the better place?